moveit2.repos

```diff
@@ -1,33 +0,0 @@
-repositories:
```

Comment:
Instead of deleting, I think it's fine keeping an empty or commented file

Reply:
I'll put it back all commented out.

Reply:
Commenting out the whole file causes vcs to return non-zero and give this output:
```
Input data is not valid format: 'NoneType' object is not subscribable
```

The way I did it here it still returns 0, it just prints this warning:
```
List of repositories is empty
```

Reply:
works for me

Reply:
It also looks good to me. I agree with Henning that we might want to use .repos in the future, if we want to merge a change depending on our upstream.